### PR TITLE
Proposal: feedback on BDD in DevOps

### PR DESCRIPTION
**Pull request by heeenkie**
Addressed PR: #1032

### CODE REVIEW

File: contributions/feedback/heeenkie/README.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Feedback on BDD in DevOps #
+
+# Members #
+Andreas Henriksson (anhenri@kth.se), Github: [heeenkie](https://github.com/heeenkie)
+
+#1032
```
Comment (Deee92) [inline contributions/feedback/heeenkie/README.md]: Please link to the PR like so:
```
[#1032](https://github.com/KTH/devops-course/pull/1032)
```

**Review by Deee92 — CHANGES_REQUESTED** [review]

File: contributions/feedback/heeenkie/README.md (hunk 1 above)
Comment (heeenkie) [reply contributions/feedback/heeenkie/README.md]: @Deee92 Done.

**Comment by Deee92** [comment]
Thanks, merging! Good luck. :)

**Review by cesarsotovalero — COMMENTED** [review]
LGTM!

**Review by Deee92 — COMMENTED** [review]
Forgot to approve before merging!

**Comment by bbaudry** [comment]
@heeenkie please update the folder name with your KTH id (anhenri) instead of your github id
thanks!

**Pull request merged**